common/module.mk

```diff
@@ -52,7 +52,8 @@ MODULE_OBJS := \
 	winexe_ne.o \
 	winexe_pe.o \
 	xmlparser.o \
-	zlib.o
+	zlib.o \
+	text-to-speech.o
```

Comment:
This list is mostly alphabetical, so it would probably make more sense to move it just after `system.o`.